libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/Level.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing;
+
+import java.util.Arrays;
+
+/**
+ * The Level class defines a set of standard tracing levels that can be used to control tracing output.
+ * The tracing Level objects are ordered and are specified by ordered integers.
+ * Enabling tracing at a given level also enables tracing at all higher levels.
+ *
+ * Levels in descending order are
+ * <ul>
+ *  <li>ROOT(highest value)</li>
+ *  <li>TERSE</li>
+ *  <li>INFO</li>
+ *  <li>DEBUG</li>
+ *  <li>TRACE(lowest value)</li>
+ * </ul>
+ *
+ * In addition, there is a level DISABLED that can be used to turn off tracing. By default, tracing is disabled.
+ *
+ */
+public enum Level {
+    /**
+     * DISABLED is a special level that can be used to turn off tracing. This level is initialized to Integer.MAX_VALUE.
+     */
+    DISABLED(Byte.MAX_VALUE),
+
+    /**
+     * ROOT is a tracing level indicating top level/root spans.
+     */
+    ROOT((byte) 100),
```

Comment:
This level also does not make sense (to me), the top level spans should have no parent, there is no need to have a dedicated level for it

Reply:
This level is basically for top level span say http request. This could be useful in usecases where we would just want details(latency, resource consumption etc) for the entire request.

I am not sure if the comments was helpful in making clear the level usage. So let me try here:

The code path could have instrumentation with different levels (ROOT, INFO, DEBUG etc.). Only those spans would be created whose level is same or more than the configured tracing level in the system.

For example lets say we have set tracing level as INFO. That means only the instrumented spans with INFO, TERSE and ROOT levels will be created and all other spans will be ignored through no-op spans.

Reply:
> This level is basically for top level span say http request. This could be useful in usecases where we would just want details(latency, resource consumption etc) for the entire request.

This is artificial and confusing level I think, I am looking it at from the user perspective and nothing prevent me to use this level anywhere. Other levels at least have some meaning.

Reply:
I am somehow not getting aligned on this change. Let me seek other's feedback on this

Reply:
Levels are removed for now